modules/services/agentless-scanning/main.tf

```diff
@@ -50,6 +50,31 @@ data "aws_iam_policy_document" "scanning" {
     ]
   }
 
+  statement {
+    sid = "AllowKMSEncryptDecrypt",
+
+    action = [
+      "kms:DescribeKey",
+      "kms:Encrypt",
+      "kms:Decrypt",
+      "kms:ReEncrypt*",
+      "kms:GenerateDataKey*",
+      "kms:CreateGrant"
+    ],
+
+    resource = [
```

Comment:
Terraform Data block `aws_iam_policy_document` has argument `resources` instead of `resource`.

Reply:
fixed